src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,22 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    // if (fromIndex < 0)  {
+    //   fromIndex = this.length + fromIndex;
+    // }
+
+    // if (fromIndex < 0) {
+    //   fromIndex = 0;
+    // }
+
```

Comment:
I added this comments, because i think these operators are necessary for checking on the index '-' , but tests pass in both cases